src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/View/addresses.html.twig

```diff
@@ -92,12 +92,15 @@
                     ></a>
 
                     <div class="dropdown-menu dropdown-menu-right">
-                      <a class="btn tooltip-link dropdown-item"
-                         href="{{ getAdminLink('AdminAddresses', true, {'id_address': address.addressId, 'deleteaddress': 1, 'back': app.request.uri}) }}"
+                      <button type="button"
+                              class="btn tooltip-link dropdown-item js-form-submit-btn"
+                              data-form-confirm-message="{{ 'Are you sure you want to delete the selected item(s)?'|trans({}, 'Admin.Global') }}"
```

Comment:
Currently, there are two exceptions on this page: one when deleting one row (_Delete selected item?_) and another when deleting in bulk (_Delete selected items?_). Which exception does this PR fix? Or is it a 2-in-1 exception that would work for both cases?

Reply:
this confirm message I updated is for single row deletion

Reply:
Okay, thanks @sowbiba! In this case, I'd keep it consistent with the other exception (the bulk action exception) and write _Delete selected item?_, localized in _Admin.Notifications.Warning_. It is not a deletion modal because the page has not been migrated in Symfony yet, right?

Reply:
didn't want to introduice this subject :joy:  but yes, this should be a modal, not a simple confirm box

Reply:
And in the other modals for deletion we did, the confirm message was "Are you sure you want to "{{ 'Are you sure you want to delete the selected item(s)?'|trans({}, 'Admin.Global') }}"
That's why I choosed this wording

Reply:
I know, and you're right about it. But since the bug is about a confirm box, we should keep the behavior iso, at least for now, and turn these two confirm boxes into one modal when the page will be migrated.

Reply:
Actually, the page is migrated for 1.7.7 and I've just come across this PR, https://github.com/PrestaShop/PrestaShop/pull/18464, so I guess it should definitely be a modal here. It looks like @marionf is the owner of this topic, can you confirm?

Reply:
The PR #18464 is for bulk action

https://github.com/PrestaShop/PrestaShop/issues/17847 :thinking:

Reply:
We keep the classic confirm for this PR. And to avoid conflicts, the modal will be done in another PR

Reply:
:+1: